src/App.jsx

```diff
@@ -1,8 +1,10 @@
 import './App.css';
+import Login from './component/Login/Login';
 
 const App = () => (
   <>
-    <h1>This is the beginning of something.</h1>
+    <Login />
```

Comment:
I dont think this makes sense, does it? To have the login component render in the app?